### DISCUSSION TITLE
How do I learn programming?

### DISCUSSION THREAD

**Discussion by willzyx-hub** [compnerd/swift-win32#759] [General]
Hello, so recently i found swift-win32 that allow to create GUI Swift Windows apps. now i wan't to create my own GUI apps but i'm really blind to Programming so how do i learn some and create one ? i mean is there really a tutorial for it ?

**Comment by compnerd** [comment]
I don't think that this is the proper place to general programming content. I would recommend that you check on the Swift forums for general programming content. As to a tutorial for this framework, that is something that I would be open to. There are some basic docs in the docs directory, but the addition of a tutorial built with the DocC framework would be interesting once the DocC support for Windows is complete.
